tests/pytorch/test_onnx_export.py

```diff
@@ -328,6 +328,57 @@ def get_attn_mask_str(use_mask, attn_mask_type):
     return attn_mask_str
 
 
+class TestFP8_GEMM(nn.Module):
```

Comment:
PyTest will automatically interpret objects starting with `Test` or `test_` as tests. A quick fix is to rename it to something like `_TestFP8_GEMM`. However, now that the class is no longer is a local scope, I'd prefer a more descriptive name like `FP8GemmModule` or `CustomFP8Gemm`:
```suggestion
class FP8GemmModule(nn.Module):
```